src/types/Post.tsx

```diff
@@ -0,0 +1,11 @@
+import { User } from './User';
+import { Comment } from './Comment';
+
+export interface Post {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+  user?: User | null,
```

Comment:
There is no way, the user could be `null`
```suggestion
  user?: User,
```